edvart/report.py

```diff
@@ -369,7 +369,7 @@ def add_overview(
         self.sections.append(
             Overview(
                 subsections=subsections,
-                verbosity=verbosity if verbosity is not None else self.verbosity,
+                verbosity=verbosity or self.verbosity,
```

Comment:
This does not really since the value of `verbosity.LOW` is `0`, which is falsy.
```python
>>> from edvart import Verbosity
>>> Verbosity.LOW or "something else"
'something else'
```

Reply:
Indeed. I created a PR with braking change of making all verbosities positive: https://github.com/datamole-ai/edvart/pull/84. Will get back to this PR once it's merged.

Reply:
#84 merged. This PR is ready from my side.